test/jdk/javax/net/ssl/sanity/ciphersuites/SystemPropCipherSuitesOrder.java

```diff
@@ -81,6 +83,7 @@ public class SystemPropCipherSuitesOrder extends SSLSocketTemplate {
     private static String[] clientcipherSuites;
 
     public static void main(String[] args) {
+        SecurityUtils.removeFromDisabledTlsAlgs("TLS_RSA_*");
```

Comment:
A suggestion - re-enable TLS_RSA only if you need to. You could check if `servercipherSuites` or `clientciphersuites` starts with "TLS_RSA".

Reply:
Done